react/components/ListsPage/Header.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { Component, ReactNode } from 'react'
+import { FormattedMessage, InjectedIntlProps, injectIntl, IntlShape } from 'react-intl'
+import { IconPlusLines } from 'vtex.styleguide'
+
+import ApolloClient from 'apollo-client'
+import { withApollo, WithApolloClient } from 'react-apollo'
+
+import DialogMessage from '../Dialog/DialogMessage'
+import CreateList from '../Form/CreateList'
+import UpdateList from '../Form/UpdateList'
+import MenuOptions from '../MenuOptions/MenuOptions'
+
+import { deleteList } from '../../GraphqlClient'
+
+interface HeaderState {
+  showcreateList?: boolean
+  showUpdateList?: boolean
+  showDeleteConfirmation?: boolean
+}
+
+interface HeaderProps {
+  isDefault?: boolean
+  list?: List
+  intl?: IntlShape
+  onListCreated: (list: List) => void
+  onListUpdated: (list: List) => void
+  onListDeleted: () => void
+  client?: ApolloClient<any>
+}
+
+const ICONS_SIZE = 20
+
+class Header extends Component<HeaderProps & InjectedIntlProps & WithApolloClient<any>, HeaderState> {
+  public state: HeaderState = {}
+  private options: Option[] = [
```

Comment:
is this used anywhere?

Reply:
Yes, it is used by the `MenuOptions` component rendered by the `Header`